plugins/inventory/proxmox.py

```diff
@@ -114,6 +114,18 @@
   mailservers: "'mail' in (proxmox_tags_parsed|list)"
 compose:
   ansible_port: 2222
+  
+# Using the inventory to allow ansible to connect via the first IP address of the VM / Container (Default is connection by name of QEMU/LXC guests)
+# Note: The false | default('mystr') pattern is necessary due to how compose parses Jinja2. This would add the variable to every host used by the inventory
```

Comment:
Limiting the line length helps readability on devices with limited horizontal screen width:
```suggestion
# Using the inventory to allow ansible to connect via the first IP address of the VM / Container
# (Default is connection by name of QEMU/LXC guests)
# Note: The false | default('mystr') pattern is necessary due to how compose parses Jinja2.
#       This would add the variable to every host used by the inventory.
```